pyprosail/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "PyProSAIL" %}
+{% set version = "1.0.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_value = "d632d7216b2d9b64968f3b5f1dd113ab101dc6e1f6a146e28a7bc2c35e0fc268" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```

Comment:
the parsed URL must be exactly like this:
```
https://pypi.io/packages/source/p/pyprosail/PyPROSAIL-1.0.2.tar.gz
```

otherwise it won't work

Reply:
now all CIs will pass because the don't download anything

Reply:
use `{{ name|lower }}`

Reply:
`https://pypi.io/packages/source/P/PyPROSAIL/PyPROSAIL-1.0.2.tar.gz` actually works as well; you just can't mix case between the two.

Reply:
thanks @dougalsutherland, I always learn a lot from hanging around on conda-forge.. !

Reply:
...however I just noticed that that link is broken